.yamato/config.metadata

```diff
@@ -1,8 +1,5 @@
 editors:
-  - version: 2019.4
-  - version: 2020.3
-  - version: 2021.2
-  - version: trunk
+  - version: 5e2ac778e08501f066384234f79629ad6ece77ac
```

Comment:
We need to test against multiple versions of Unity. The list here keeps track of the versions that we explicitly support.

Also concerned about the replaced of a branch with just a direct version number. We want to continuously track trunk here.

Reply:
This was added to test just for me against my personal branch before it goes into trunk. I will put back the other versions when my branch goes into trunk, and will revert this at the same time